cluster-autoscaler/processors/provreq/injector.go

```diff
@@ -170,3 +204,14 @@ func key(pr *provreqwrapper.ProvisioningRequest) string {
 func (p *ProvisioningRequestPodsInjector) LastProvisioningRequestProcessTime() time.Time {
 	return p.lastProvisioningRequestProcessTime
 }
+
+// UpdateLastProcessTime updates the time we last processed a ProvisioningRequest
+// to now. This time is used to skip waiting between loops if a request
+// was processed in the last loop.
+func (p *ProvisioningRequestPodsInjector) UpdateLastProcessTime() {
+	if p == nil {
```

Comment:
I'd move this nil check outside this method, to the caller

Reply:
Done.